### PR TITLE
simplify png-to-lua output

### PR DESCRIPTION
**Pull request by mikeylove**
...and regenerated relevant resources.

### CODE REVIEW

File: reascripts/common/scripts/png_to_lua.rb [hunk 1]
```diff
@@ -19,11 +19,11 @@
...
   f.write("  height = #{image.height},\n")
 
-  f.write("  bytes = table.concat({\n")
-  image.to_blob.bytes.each_slice(100) do |slice|
-    f.write("string.char(#{slice.join(",")}),\n")
-  end
-  f.write("  })\n")
+  bytes = image.to_blob.bytes.map { |b|
+    format("\\%d", b)
+  }.join
```
Comment (mikeylove) [inline reascripts/common/scripts/png_to_lua.rb:24]: just command-tabbed my way to vscode and this was line was on my screen and i felt a sincere sense of loss that the return value of `format("\\%d")` isn't a function expecting a byte argument

Comment (ramen) [reply reascripts/common/scripts/png_to_lua.rb:24]: Yeah, Ruby's FP support is lackluster at times. There is a convenient shortcut for no-argument methods, which could be used/abused along with Ruby's open classes:

```
class Integer
  def code_string
    return format("\\%d", self)
  end
end

bytes = blob.map(&:code_string).join
```

**Review by ramen — APPROVED** [review]
:shipit:

File: reascripts/common/scripts/png_to_lua.rb (hunk 1 above)
Comment (mikeylove) [reply reascripts/common/scripts/png_to_lua.rb:24]: > could be used/**abused** along with Ruby's open classes

me at ruby jobs in the past feeling pretty seen right now

**Pull request merged**